ipatests/test_xmlrpc/test_host_plugin.py

```diff
@@ -261,6 +261,13 @@ def test_update_simple(self, host):
         res = command()['result']
         assert len(res) == 1
 
+        # test host-find with --pkey-only
+        command = host.make_find_command(
+            fqdn=host.fqdn, pkey_only=True)
```

Comment:
I tried to run the test on a system without the patch, and the test does not detect the failure. 
1- the host entry does not contain any ssh key, you should rather test with the hostname of the master
2- in the output, you need to check if there is a result[u'result'][u'ipasshpubkey'] value.

Reply:
I have made the changes as per review comments